conn.go

```diff
@@ -218,6 +237,18 @@ func (c *connect) compressBuffer(start int) error {
 }
 
 func (c *connect) sendData(block *proto.Block, name string) error {
+	if c.closed {
+		err := errors.New("attempted sending on closed connection")
+		c.debugf("[send data] err: %v", err)
+		return err
+	}
+
+	if c.buffer == nil {
```

Comment:
Is it something you run into when debugging or it's something you want to presumably safe?